workflow-job

```diff
@@ -533,6 +587,10 @@ flows["${sha1}"] = {
             def dPassword = dockerPassword();
             def tagName = tag();
             sh "docker login -u ${dUser} -p ${dPassword}"
+
+            sh "docker tag ${images.jamesCassandraRabbitMQ} linagora/james-cassandra-rabbitmq-swift:${tagName}"
+            sh "docker push linagora/james-project:${tagName}"
```

Comment:
it should be james-rabbitmq-project, no? See https://github.com/linagora/james-project/blob/master/dockerfiles/packaging/guice/cassandra/Dockerfile#L3